src/test/java/pl/psnc/dei/iiif/ConversionDataHolderTransformerTest.java

```diff
@@ -0,0 +1,426 @@
+package pl.psnc.dei.iiif;
+
+import lombok.SneakyThrows;
+import org.apache.jena.atlas.json.JSON;
+import org.apache.jena.atlas.json.JsonObject;
+import org.apache.jena.atlas.json.JsonValue;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.test.context.junit4.SpringRunner;
+import pl.psnc.dei.model.Aggregator;
+import pl.psnc.dei.model.Record;
+import pl.psnc.dei.model.conversion.ConversionData;
+import pl.psnc.dei.model.conversion.ConversionTaskContext;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+@RunWith(SpringRunner.class)
+public class ConversionDataHolderTransformerTest {
+    private final String RECORD_IDENTIFIER = "/11602/HERBARWUXWUXAUSTRIAX102495";
+
+    private final JsonObject EUROPEANA_RECORD_JSON = JSON.parse("{ \n" +
```

Comment:
I think you can just minify this to one-line literal or create separate json files as test resources. There is to much to scroll.

Reply:
well, moving this to file is reasonable, but it create some overhead, isn't it? Especially other tests uses same approach as mine.

Reply:
but moved to separate files